ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2362,6 +2362,104 @@
       "mediaType": "image/svg+xml"
     }
   },
+  {
+    "id": "nodejs-postgres",
+    "creator": "Ibrahim Jarif",
+    "name": "Nodejs and Postgres Stack",
+    "description": "Nodejs stack with Postgres",
+    "scope": "general",
+    "tags": [
+      "node.js",
+      "npm",
+      "postgres"
+    ],
+    "components": [
+      {
+        "name": "Node.js",
+        "version": "8.11.3"
+      },
+      {
+        "name": "NPM",
+        "version": "5.6.0"
+      },
+      {
+        "name": "Postgres",
+        "version": "9.6"
+      }
+    ],
+    "workspaceConfig": {
+      "defaultEnv": "default",
+      "environments": {
+        "default": {
+          "machines": {
+            "database": {
+              "attributes": {
+                "memoryLimitBytes": "547483648"
+              },
+              "servers": {
+                "postgres": {
+                  "attributes": {},
+                  "port": "5432",
+                  "protocol": "tcp"
+                }
+              },
+              "volumes": {},
+              "installers": [
+                "org.eclipse.che.exec",
+                "org.eclipse.che.terminal"
+              ],
+              "env": {
+                "POSTGRESQL_ADMIN_PASSWORD": "adminpassword",
+                "POSTGRESQL_DATABASE": "my_data",
+                "POSTGRESQL_USER": "user",
+                "POSTGRESQL_PASSWORD": "password"
+              }
+            },
+            "nodejs": {
+              "attributes": {
+                "memoryLimitBytes": "1147483648"
+              },
+              "servers": {
+                "node": {
+                  "attributes": {},
+                  "port": "8080",
+                  "protocol": "http"
+                }
+              },
+              "volumes": {},
+              "installers": [
+                "org.eclipse.che.exec",
+                "org.eclipse.che.terminal",
+                "org.eclipse.che.ls.js-ts",
+                "org.eclipse.che.ws-agent"
+              ],
+              "env": {}
+            }
+          },
+          "recipe": {
+            "type": "openshift",
```

Comment:
is it tied to openshift or could it be kubernetes (so it can work both on k8s and openshift ? )

Reply:
I think it would work on kubernetes too. But I can't be sure. i haven't tried it.